### ISSUE TITLE
Getting cyclic error in console - Error on reading console input: null

### ISSUE THREAD

**Issue by degustator** [orientechnologies/orientdb#3474]
Environment: RHEL 6.5
Right after launching console using this command:

$ /data/orientdb/orientdb-community-1.7.10/bin/console.sh

It looks like this:
orientdb> Error on reading console input: null
java.lang.NullPointerException
        at com.orientechnologies.common.console.TTYConsoleReader.readLine(TTYConsoleReader.java:74)
        at com.orientechnologies.common.console.OConsoleApplication.run(OConsoleApplication.java:127)
        at com.orientechnologies.orient.graph.console.OGremlinConsole.main(OGremlinConsole.java:55)

As it goes in cycle its unstoppable.

**Comment by lvca** [comment]
Have you tried with 2.0?

**Comment by degustator** [comment]
Yes. Got same error :(

**Comment by stuartcarnie** [comment]
I am seeing it with v2.0.1 too, but only when running console.sh on my Ubuntu 14.04 VM. If I connect via console.sh to Ubuntu OrientDB from OS X, it is fine

**Comment by stuartcarnie** [comment]
It was a permission issue. my current shell was not part of the `orientdb` group. I logged out and back in, and all was well.

**Comment by lvca** [comment]
@stuartcarnie Please could you copy here the stack trace with 2.0.1? I'd like to provide a better exception in this case.

**Comment by stuartcarnie** [comment]
```
orientdb> Error on reading console input: null
java.lang.NullPointerException
	at com.orientechnologies.common.console.TTYConsoleReader.readLine(TTYConsoleReader.java:94)
	at com.orientechnologies.common.console.OConsoleApplication.run(OConsoleApplication.java:130)
	at com.orientechnologies.orient.graph.console.OGremlinConsole.main(OGremlinConsole.java:62)
```

**Comment by lvca** [comment]
Please let me know if this is fixed in 2.0.2-SNAPSHOT and/or 2.1-SNAPSHOT.
